chef/data_bags/crowbar/migrate/network/101_add_ironic_network.rb

```diff
@@ -0,0 +1,20 @@
+def upgrade(ta, td, a, d)
+  unless a["networks"].key? "ironic"
+    a["networks"]["ironic"] = ta["networks"]["ironic"]
+  end
+
+  # copy intf1 as intf3
+  a["conduit_map"].each do |cm|
+    unless cm["conduit_list"].key? "intf3"
+      cm["conduit_list"]["intf3"] = cm["conduit_list"]["intf1"]
```

Comment:
> I'm not sure what will happen if there's no entry in the conduit_map matching the network some barclamp is trying to allocate an IP from.

It's going to fail. Latest when trying to make the allocation active with the chef-client run.

But then, using the same interfaces as admin is going to fail as well, just in very different (and probabyl hard to debug) way.